ui/src/components/executions/VarValue.vue

```diff
@@ -49,35 +49,36 @@
         id: string;
     }
 
-    const props = defineProps({
-        value: {
-            type: [String, Object, Boolean, Number],
-            required: false,
-            default: undefined
-        },
-        execution: {
-            type: Object as () => Execution,
-            required: false,
-            default: undefined
-        },
-        restrictUri: {
-            type: Boolean,
-            required: false,
-            default: false
-        }
+    interface FileMetadata {
+        size: number;
+    }
+
+    interface Props {
+        value?: string | object | boolean | number;
+        execution?: Execution;
+        restrictUri?: boolean;
+    }
+
+    const props = withDefaults(defineProps<Props>(), {
+        value: undefined,
+        execution: undefined,
+        restrictUri: false,
```

Comment:
One small suggestion and we can merge

```suggestion
    const props = withDefaults(defineProps<{
        value?: string | object | boolean | number;
        execution?: Execution;
        restrictUri?: boolean;
    }>(), {
        value: undefined,
        execution: undefined,
        restrictUri: false,
```